playbook-website/app/views/pages/code_snippets/positioning_jsx.txt

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+const PositioningDefault = () => {
+  return (
+   <User
+        align="center"
+        avatarUrl="https://randomuser.me/api/portraits/women/44.jpg"
+        name="Anna Black"
+        orientation="vertical"
+        size="lg"
+        title="Remodeling Consultant"
+        zIndex={10}
```

Comment:
Let's use a shorter example. Perhaps a badge or pill. anything that is short and we can see zIndex right away.

Reply:
got it, i'll be working on all the changes this morning

Reply:
I figured since the zIndex layout was changed by Dat that was how you wanted it but I will have that done today